docs/tables/aws_efs_file_system.md

```diff
@@ -11,6 +11,7 @@ select
   name,
   file_system_id,
   owner_id,
+  automatic_backup,
```

Comment:
Will be good if we add a separate example to check the `Enable automatic backups`.